extensions/library-genesis/README.md

```diff
@@ -0,0 +1,5 @@
+# Library Genesis
+
+Search books on Library Genesis and view them.
+
+![Screencast](./screencasts/library-genesis-1.png)
```

Comment:
Sorry @yz3440 I just noticed that you used the image in the screencasts folder in your README. for images used in the README you should place them in the `assets` folder. So one last thing to would be to add this image to your existing `assets` folder